.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+	"recommendations": ["esbenp.prettier-vscode"]
```

Comment:
This is for VS Code users :)

Reply:
This will show this banner if the recommended extension is not installed
![image](https://user-images.githubusercontent.com/1477453/70069515-8ffdbe80-15fa-11ea-8ff5-f159e90eee2b.png)

Reply:
There we can install all or we can press `Show Recommendations` which will display:
![image](https://user-images.githubusercontent.com/1477453/70069606-b9b6e580-15fa-11ea-875a-5ca253f39d01.png)

Reply:
nice